src/extensions/clusters/clusters.py

```diff
@@ -28,11 +29,17 @@ def createjob_individual_cluster_features(labbox, recording_object, sorting_obje
             unit_id=unit_id
         )
 
-@hi.function('individual_cluster_features', '0.1.2')
+def subsample_inds(n, m):
+    if m >= n:
+        return range(n)
+    incr = n / m
+    return [int(math.floor(i * incr)) for i in range(m)]
+
+@hi.function('individual_cluster_features', '0.1.3')
 @hi.container('docker://magland/labbox-ephys-processing:0.3.19')
 @hi.local_modules([os.getenv('LABBOX_EPHYS_PYTHON_MODULE_DIR')])
 @le.serialize
-def individual_cluster_features(snippets_h5, unit_id):
+def individual_cluster_features(snippets_h5, unit_id, max_num_events=1000):
```

Comment:
Limit number of events to a maximum of 1000 for cluster features. Eventually we will want this to be a parameter in the GUI